client/state/action-types.ts

```diff
@@ -823,15 +823,22 @@ export const SIGNUP_PROGRESS_PROCESS_STEP = 'SIGNUP_PROGRESS_PROCESS_STEP';
 export const SIGNUP_PROGRESS_REMOVE_STEP = 'SIGNUP_PROGRESS_REMOVE_STEP';
 export const SIGNUP_PROGRESS_SAVE_STEP = 'SIGNUP_PROGRESS_SAVE_STEP';
 export const SIGNUP_PROGRESS_SUBMIT_STEP = 'SIGNUP_PROGRESS_SUBMIT_STEP';
-export const SIGNUP_STEPS_SITE_INFO_COLLECTION_UPDATE = 'SIGNUP_STEPS_SITE_INFO_COLLECTION_UPDATE';
-export const SIGNUP_STEPS_SITE_INFO_UPDATE_CURRENT_INDEX =
-	'SIGNUP_STEPS_SITE_INFO_UPDATE_CURRENT_INDEX';
 export const SIGNUP_STEPS_DESIGN_TYPE_SET = 'SIGNUP_STEPS_DESIGN_TYPE_SET';
 export const SIGNUP_STEPS_SITE_GOALS_SET = 'SIGNUP_STEPS_SITE_GOALS_SET';
 export const SIGNUP_STEPS_SITE_TITLE_SET = 'SIGNUP_STEPS_SITE_TITLE_SET';
 export const SIGNUP_STEPS_SITE_TYPE_SET = 'SIGNUP_STEPS_SITE_TYPE_SET';
 export const SIGNUP_STEPS_SITE_VERTICAL_SET = 'SIGNUP_STEPS_SITE_VERTICAL_SET';
 export const SIGNUP_STEPS_SURVEY_SET = 'SIGNUP_STEPS_SURVEY_SET';
+
+// Signup DIFM Actions
+export const SIGNUP_STEPS_SITE_INFO_COLLECTION_UPDATE = 'SIGNUP_STEPS_SITE_INFO_COLLECTION_UPDATE';
+export const SIGNUP_STEPS_SITE_INFO_UPDATE_CURRENT_INDEX =
+	'SIGNUP_STEPS_SITE_INFO_UPDATE_CURRENT_INDEX';
+
+export const SIGNUP_STEPS_WEBSITE_CONTENT_UPDATE = 'SIGNUP_STEPS_WEBSITE_CONTENT_UPDATE';
+export const SIGNUP_STEPS_WEBSITE_CONTENT_UPDATE_CURRENT_INDEX =
+	'SIGNUP_STEPS_WEBSITE_CONTENT_UPDATE_CURRENT_INDEX';
+
```

Comment:
Moved the DIFM actions here so that we can sort of create some sort of cohesion for the actions :)